internal/server/server.go

```diff
@@ -49,7 +49,28 @@ func RunServer(config *Config) error {
 	logger := log.NewCtxLogger(config.LogLevel, []string{domain.TraceIDKey})
 	crypto := crypto.NewAES(config.EncryptionSecretKeyKey)
 	validator := validator.New()
-	notifier, err := notifiers.NewClient(&config.Notifier, logger)
+	notifierConfig := notifiers.ConfigMultiClient{}
+	if &config.Notifiers != nil {
+		notifierConfig = config.Notifiers
+
+	} else {
+		// map old to the new format
+		var provider = config.Notifier.Provider
+		configSlack, err := notifiers.NewSlackConfig(&config.Notifier)
+		if err != nil {
+			return err
+		}
+		for n, workspace := range configSlack.Workspaces {
+			notifier_ := notifiers.Notifier{
+				Provider:    provider,
+				AccessToken: workspace.AccessToken,
+				Criteria:    "true",
```

Comment:
with the current approach this would work since slack handles criteria on its own, ideally criteria should be handled by the notifier manager before calling each notifier's `Notify()`.
related comment: https://github.com/goto/guardian/pull/163#discussion_r1680772162

Reply:
yeah that would need config/workspace with NotifyManager, Is its not p0 we can do later. and let me explicitly add criteria eval for lark right away